packages/@office-iss/react-native-win32/src/Libraries/Components/AccessibilityInfo/AccessibilityInfo.win32.js

```diff
@@ -4,31 +4,57 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @format
  * @flow strict-local
+ * @format
  */
 
 import RCTDeviceEventEmitter from '../../EventEmitter/RCTDeviceEventEmitter';
-import NativeAccessibilityInfo from './NativeAccessibilityInfo';
-import type {EventSubscription} from 'react-native/Libraries/vendor/emitter/EventEmitter';
-import type {HostComponent} from '../../Renderer/shims/ReactNativeTypes';
 import {sendAccessibilityEvent} from '../../Renderer/shims/ReactNative';
+import type {HostComponent} from '../../Renderer/shims/ReactNativeTypes';
+import Platform from '../../Utilities/Platform';
+import type EventEmitter from '../../vendor/emitter/EventEmitter';
+import type {EventSubscription} from '../../vendor/emitter/EventEmitter';
+import NativeAccessibilityInfo from './NativeAccessibilityInfo';
+import NativeAccessibilityManagerIOS from './NativeAccessibilityManager';
 import legacySendAccessibilityEvent from './legacySendAccessibilityEvent';
-import {type ElementRef} from 'react';
+import type {ElementRef} from 'react';
 
-const REDUCE_MOTION_EVENT = 'reduceMotionDidChange';
-const TOUCH_EXPLORATION_EVENT = 'touchExplorationDidChange';
+// Events that are only supported on iOS.
+type AccessibilityEventDefinitionsIOS = {
+  announcementFinished: [{announcement: string, success: boolean}],
+  boldTextChanged: [boolean],
+  grayscaleChanged: [boolean],
+  invertColorsChanged: [boolean],
+  reduceTransparencyChanged: [boolean],
+};
 
 type AccessibilityEventDefinitions = {
+  ...AccessibilityEventDefinitionsIOS,
+  change: [boolean], // screenReaderChanged
   reduceMotionChanged: [boolean],
   screenReaderChanged: [boolean],
-  // alias for screenReaderChanged
-  change: [boolean],
 };
 
-type AccessibilityEventTypes = 'focus' | 'click';
+type AccessibilityEventTypes = 'click' | 'focus';
 
-const _subscriptions = new Map();
+// Mapping of public event names to platform-specific event names.
+const EventNames: Map<$Keys<AccessibilityEventDefinitions>, string> =
+  Platform.OS === 'android' || Platform.OS === 'windows'
```

Comment:
I assume we coded this since this is a fork, would it make more sense to make the check `not apple`?

Reply:
and flip the blocks? 
Note: Only asking :) And if so, there are a few cases below as well.

Reply:
The original was to just check "if android". Windows and android have the same behavior for this file so swapping the blocks instead of adding the check for Windows will work! Just depends on if there's a preference for the file to match as close to upstream as possible.

Reply:
I worry a little about how well the diffing algorithm will notice changes in the moved block and apply them.  Whereas modifying the if, the code stays in the same place so the diffing logic is more likely to correctly update the code as changes come from core.

Reply:
That makes total sense, sorry for the noise. 
Should we consider porting this change to FB? So perhaps we don't have to do anything in the future?